internal/broker/broker.go

```diff
@@ -540,11 +541,18 @@ func (b *Broker) handleIsAuthenticated(ctx context.Context, session *session, au
 		}
 		expiryCtx, cancel := context.WithDeadline(ctx, response.Expiry)
 		defer cancel()
+
+		// The default interval is 5 seconds, which means the user has to wait up to 5 seconds after
+		// successful authentication. We're reducing the interval to 1 second to improve UX a bit.
+		response.Interval = 1
```

Comment:
This is quite a big reduction in the polling time. Did you test it manually to see if it doesn't break anything (particularly in the UI)?